data_juicer/ops/mapper/video_audio_speech_ASR_mapper.py

```diff
@@ -0,0 +1,105 @@
+import librosa
+from data_juicer.utils.mm_utils import extract_audio_from_video
+from data_juicer.utils.model_utils import get_model, prepare_model
+from ..base_op import OPERATORS, Mapper
+import gc
+from data_juicer.utils.constant import Fields, MetaKeys
+
+OP_NAME = 'video_audio_speech_ASR_mapper'
+
+import torch
+torch.set_num_threads(1)
+
+
+@OPERATORS.register_module(OP_NAME)
+class VideoAudioSpeechASRMapper(Mapper):
```

Comment:
`ASR` actually includes "Speech". So the name can be VideoAudioASRMapper, video_audio_ASR_mapper